tests/app/test_jinja_filters.py

```diff
@@ -28,6 +28,108 @@
 from app.questionnaire.questionnaire_schema import QuestionnaireSchema
 from app.utilities.schema import load_schema_from_name
 
+TEST_FORMAT_CURRENCY_PARAMS = (
+    "value, currency, decimal_limit, formatted_currency",
+    [
+        (Decimal("3000.44545"), "GBP", None, "£3,000"),
+        (Decimal("2.1"), "GBP", None, "£2"),
+        (Decimal("3000"), "GBP", 0, "£3,000"),
+        (Decimal("3000"), "JPY", 0, "JP¥3,000"),
+        (Decimal("2.1"), "GBP", 1, "£2.1"),
+        (Decimal("123.45"), "GBP", 1, "£123.4"),
+        (Decimal("2.1"), "GBP", 1, "£2.1"),
+        (11, "GBP", 2, "£11.00"),
+        (0, "GBP", 2, "£0.00"),
+        (0.00, "GBP", 2, "£0.00"),
+        (11000, "USD", 2, "US$11,000.00"),
+        (11000, "USD", 2, "US$11,000.00"),
+        (Decimal("11.99"), "GBP", 2, "£11.99"),
+        (2, "GBP", 6, "£2.00"),
+        (Decimal("1.1"), "GBP", 6, "£1.10"),
+        (Decimal("1.10"), "GBP", 6, "£1.10"),
+        (Decimal("1.100"), "GBP", 6, "£1.100"),
+        (Decimal("1.1000"), "GBP", 6, "£1.1000"),
+        (Decimal("2.14564"), "GBP", 6, "£2.14564"),
+        (Decimal("1.00000"), "GBP", 6, "£1.00000"),
+        (Decimal("3000.445"), "GBP", 6, "£3,000.445"),
```

Comment:
Minor, but like the ones below, it would be clearer if this was organise by like input length currency type etc.